udata_front/theme/gouvfr/datagouv-components/src/helpers/index.ts

```diff
@@ -39,14 +39,14 @@ export const filesize = (val: number) => {
   return `${val.toFixed(1)}Y${suffix}`
 }
 
-export const summarize = (val: number) => {
+export const summarize = (val: number, fractionDigits = 0) => {
```

Comment:
This was the test I did to check that the modification is showing on udata-front